### ISSUE TITLE
Hot Reloading For Js or Jsx Files

### ISSUE THREAD

**Issue by ahgsql** [remotion-dev/remotion#445]
Hi, when working with JS or JSX components, hot reloading is not working.
As soon as i chaned extension to tsx its working again. 
Hot reloader should watch jsx iles also, please implement this.

**Comment by tomByrer** [comment]
`tsconfig.json`
```json
  "compilerOptions": {
    "allowJs": true,
    "checkJs": true,
 ...
 ```
 https://www.typescriptlang.org/tsconfig#compilerOptions
 
 Does that work for you?

**Comment by ahgsql** [comment]
> `tsconfig.json`
> 
> ```json
>   "compilerOptions": {
>     "allowJs": true,
>     "checkJs": true,
>  ...
> ```
> 
> https://www.typescriptlang.org/tsconfig#compilerOptions
> 
> Does that work for you?

nope, its not worked.

**Comment by JonnyBurger** [comment]
Thanks for the report! I think I found a fix, can you try v2.1.10? Sorry for the missing feature!

**Comment by ahgsql** [comment]
updated and working like charm Jonny, Thx!
